content/en/docs/kubernetes/helm.md

```diff
@@ -0,0 +1,7 @@
+---
+title: OpenTelemetry Helm Charts
+weight: 3
+spelling: cSpell:ignore Otel
+---
+
+Coming Soon!
```

Comment:
~~~suggestion
You can add the [OpenTelemetry Helm Charts](https://github.com/open-telemetry/opentelemetry-helm-charts) repository using the [Helm](https://helm.sh/) CLI as follows:

```sh
helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
```
~~~